### PR TITLE
added avneesh0612

### PR DESCRIPTION
**Pull request by avneesh0612**
I have added my profile :D

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/710"></a>

### CODE REVIEW

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Can you revert the package lock file?

**Comment by avneesh0612** [comment]
Oh okay

**Comment by avneesh0612** [comment]
I forgot the command 🤦‍♂️🤦‍♂️

**Comment by avneesh0612** [comment]
Reverted it @krishguptadev

File: public/data/avneesh0612.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      "url": "https://github.com/avneesh0612",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/avneesh0612",
+      "icon": "twitter"
+    },
+    {
+      "name": "Join my discord server",
```
Comment (krishguptadev) [inline public/data/avneesh0612.json:17]:
```suggestion
      "name": "Join my Discord Server",
```

**Comment by avneesh0612** [comment]
Done @krishguptadev

**Review by krishguptadev — APPROVED** [review]

**Review by schmelto — APPROVED** [review]
Lgtm

**Review by rohan-kulkarni-25 — APPROVED** [review]
Looks Good to me !!

**Comment by avneesh0612** [comment]
Awesome! Thanks :D

File: public/data/avneesh0612.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Avneesh Agarwal",
+  "bio": "I am a web developer, designer, content creator, open source contributor and student.",
```
Comment (Panquesito7) [inline public/data/avneesh0612.json:3]: Missing comma there. Going to make PR for it. 🙂
```suggestion
  "bio": "I am a web developer, designer, content creator, open source contributor, and student.",
```

**Comment by avneesh0612** [comment]
Oh okay

**Pull request merged**